exercises/nucleotide-count/cases_test.go

```diff
@@ -0,0 +1,39 @@
+package dna
+
+// Source: exercism/problem-specifications
+// Commit: bbdcaee nucleotide-count: fix wrong order introduced in #951
+// Problem Specifications Version: 1.2.0
+
+// count all nucleotides in a strand
+var testCases = []struct {
+	description   string
+	strand        string
+	expected      map[string]int
+	errorExpected bool
+}{
+	{
+		description: "empty strand",
+		strand:      "",
+		expected:    map[string]int{"A": 0, "C": 0, "G": 0, "T": 0},
```

Comment:
Would it be easy enough to use this expected type ?
> expected      map[byte]int

and then generate this as
> expected:    map[byte]int{'A': 0, 'C': 0, 'G': 0, 'T': 0},

A nucleotide is one single letter after all.  Perhaps too nit-picky, but it seems to match the exercise problem better.
What do you think ?

Reply:
Seems reasonable to me. Always a fan of more content in hints.md as well.